src/components/sections/Events.jsx

```diff
@@ -14,48 +23,120 @@ import {SwiperSlide} from "swiper/react";
 const Events = () => {
     const eventsData = [
         {
-            id: 0,
-            name: 'Devcon',
+              id: 0,
+            name: 'Devconnect',
+            date: 'November 2023',
+            image: devconnect2Image,
+            location: 'Istanbul, Türkiye',
+            description: 'Solidity Summit, Tooling & Interoperability, UX, and Remix Challenge',
+        },
+        {
+              id: 1,
+            name: 'SmartCon by Chainlink',
+            date: 'October 2023',
+            image: smartconImage,
+            location: 'Barcelona, Spain',
+            description: 'ZKZap! Remix on Cross-Chain Circuits',
+        },
+        {
+            id: 2,
+            name: 'ETHAccra',
+            date: 'September 2023',
+            image: ethAccraImage,
+            location: 'Accra, Ghana',
+            description: 'Zero To DApp Day',
+        },
+        {
+            id: 3,
+            name: 'Web3 Lagos Conference',
+            date: 'September 2023',
+            image: web3LagosImage,
+            location: 'Lagos, Nigeria',
+            description: 'Remix for Hackathons',
+        },
+        {
+            id: 4,
+            name: 'DSRV Builder House',
+            date: 'September 2023',
+            image: dsrvImage,
+            location: 'Seoul, South Korea',
+            description: 'WELLDONE with Remix and Remix Challenge',
+        },
+        {
+            id: 5,
+            name: 'Ethcon Korea',
+            date: 'September 2023',
+            image: ethconKoreaImage,
+            location: 'Seoul, South Korea',
+            description: 'Remix Features Every Dev Should Know',
+        },
+        {
+            id: 6,
+            name: 'EthCC 6',
+            date: 'July 2023',
+            image: ethcc6Image,
+            location: 'Paris, France',
+            description: 'Remix Features Every Dev Should Know',
+        },
+        {
+            id: 7,
+            name: 'ETHDenver',
+            date: 'March 2023',
+            image: ethDenver2Image,
+            location: 'Denver, USA',
+            description: 'Remix for Hackathons',
+        },
+        {
+            id: 8,
+            name: 'ETHIndia',
+            date: 'December 2022',
+            image: ethIndiaImage,
+            location: 'Bengaluru, India',
+            description: 'Remix for Hackathons and Remix Challenge',
+        },
+        {            
+            id: 9,
+            name: 'Devcon 6',
             date: 'October 2022',
             image: devconImage,
-            location: 'Bogota, Columbia',
-            description: 'Come and See!',
+            location: 'Bogota, Colombia',
+            description: 'What's New in Remix and Remix Rewards Launch',
```

Comment:
you probably need to escape the ' in What's like this:
'What\'s New in Remix and Remix Rewards Launch'